components/ui/button/floatingButton/FloatingButton.stories.tsx

```
@@ -0,0 +1,51 @@
+import type { Meta, StoryObj } from "@storybook/react";
```

ESLint

[error] Do not import renderer package "@storybook/react" directly. Use a framework package instead (e.g. `@storybook/nextjs`, `@storybook/react-vite`, `@storybook/nextjs-vite`, `@storybook/react-webpack5`, `@storybook/react-native-web-vite`).

(storybook/no-renderer-packages)